app/views/incomes/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1 align="center"> All Income</h1>
+<p>
+  <%= link_to "Create new Income", new_income_path %>
+</p>
+<table class="table">
+  <thead>
+    <tr>
+      <th scope="col">Income</th>
+      <th scope="col">Edit</th>
+      <th scope="col">Delete</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @incomes.each do |income| %>
+    <tr>
+      <td><%= income.amount %> </td>
+      <td> <%= link_to "Edit Income", edit_income_path(income) %></td>
+      <td><%= link_to 'Delete', income_path(income), method: :delete, data: {confirm: "Are you sure?"} %></td>
+    </tr>
+    <% end %>
+    <hr>
+    <td><%= current_user.incomes.total_value %></td>
```

Comment:
don't do it like this
 rather do it like 

`current_user.totol_income`

create a method in user.rb `total_income`

def total_income
   incomes.pluck(:amount).sum
end